Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Use Node.js 20 as the base image
+FROM node:20
+
+# Set the working directory inside the container
+WORKDIR /app
+
+# Copy package.json and package-lock.json to the container
+COPY package*.json ./
+
+# Install dependencies
+RUN npm install
```

Comment:
Update this for production

Reply:
Ensure dev dependencies are not installed

Reply:
Updated this based on this document.
https://www.tomray.dev/nestjs-docker-production